vaadin-date-picker-flow-parent/vaadin-date-picker-flow/src/test/java/com/vaadin/flow/component/datepicker/DatePickerTest.java

```diff
@@ -27,6 +27,8 @@
 import com.vaadin.flow.component.shared.HasAllowedCharPattern;
 import com.vaadin.flow.component.shared.HasTooltip;
 import com.vaadin.flow.component.datepicker.DatePicker.DatePickerI18n;
+import com.vaadin.flow.component.Tag;
+import com.vaadin.flow.component.Text;
```

Comment:
These imports are needed in #4463, somehow I added them to the wrong PR by accident 🙈